configs/11.0/packages/liburcu/sources

```diff
@@ -21,7 +21,7 @@
 
 ATSRC_PACKAGE_NAME="Userspace RCU"
 ATSRC_PACKAGE_VER=0.10.0
```

Comment:
Update this to 0.10.1.  Source: http://git.lttng.org/?p=userspace-rcu.git;a=commit;h=9353ac3ca481f1b56279f09a1bd51249000994c1